src/components/ButtonPanel.js

```diff
@@ -1,39 +1,52 @@
-import PropTypes from "prop-types";
-import Big from "big.js";
+import { Component } from 'react';
+import PropTypes from 'prop-types';
 import Button from "./Button";
 
-export default function ButtonPanel () {
-  return (
-    <div>
-      <div>
-        <Button name={"AC"} />
-        <Button name={"+/-"} />
-        <Button name={"%"} />
-        <Button name={"÷"} />
-      </div>
-      <div>
-        <Button name={"7"} />
-        <Button name={"8"} />
-        <Button name={"9"} />
-        <Button name={"X"} />
-      </div>
-      <div>
-        <Button name={"4"} />
-        <Button name={"5"} />
-        <Button name={"6"} />
-        <Button name={"-"} />
-      </div>
-      <div>
-        <Button name={"1"} />
-        <Button name={"2"} />
-        <Button name={"3"} />
-        <Button name={"+"} />
-      </div>
+export default class ButtonPanel extends Component {
```

Comment:
* [ ] convert this to functional component.
 `export default ButtonPanel (props)` or `export default ButtonPanel ({data, processInput})`